### PR TITLE
Translation de

### PR DESCRIPTION
**Pull request by effjot**
I have translated the glossary entries for CC+ODC licenses, roles, categories, frequency and confidentiality, as well as a few link.type entries to German.

I also added the German Data Licence (for German government open data), commit 2cc81da.

We might need to discuss commit 4653876, which isn’t strictly a translation, but a new concept. We often get data from clients that is intended only for use in the projects contracted by them.

### CODE REVIEW

**Comment by Gustry** [comment]
Hi @effjot 

Thanks a lot for your contributions.

Can you rebase on top of master ? You would need to edit `pg_metadata/install/sql/upgrade/upgrade_to_1.1.0.sql` to make the test passing.
Otherwise, existing pgmetadata database won't have your translations if they upgrade their database from 1.0.0.

Let me know if you have any questions.

File: pg_metadata/install/sql/upgrade/upgrade_to_1.1.0.sql [hunk 1]
```diff
@@ -137,4 +137,92 @@ COMMENT ON FUNCTION pgmetadata.update_table_comment_from_dataset() IS 'Update th
...
+WHERE g.field = t.field AND g.code = t.code;
+
+DROP TABLE pgmetadata.t_glossary;
+
+
+-- new terms
+
+INSERT INTO pgmetadata.glossary (id, field, code, label_en, description_en, item_order, label_fr, description_fr, label_it, description_it, label_es, description_es, label_de, description_de) VALUES
+(130, 'dataset.license', 'dl-de/by-2-0', 'Data licence Germany – attribution – version 2.0', NULL, 80, NULL, NULL, NULL, NULL, NULL, NULL, 'Datenlizenz Deutschland – Namensnennung – Version 2.0', NULL),
+(131, 'dataset.license', 'proj', 'restricted use for project-related work', NULL, 90, NULL, NULL, NULL, NULL, NULL, NULL, 'nur für Projektbearbeitung', NULL)
```
Comment (Gustry) [inline pg_metadata/install/sql/upgrade/upgrade_to_1.1.0.sql:210]:
```suggestion
(131, 'dataset.license', 'proj', 'Restricted use for project-related work', NULL, 90, NULL, NULL, NULL, NULL, NULL, NULL, 'nur für Projektbearbeitung', NULL)
```

I will let you manage capital letters in German, but I think in English, we can ;-)

File: pg_metadata/install/sql/pgmetadata/90_GLOSSARY.sql [hunk 2]
```diff
@@ -1,43 +1,51 @@
...
 INSERT INTO pgmetadata.glossary (id, field, code, label_en, description_en, item_order, label_fr, description_fr, label_it, description_it, label_es, description_es, label_de, description_de) VALUES (12, 'link.mime', 'csv', 'text/csv', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL);
+INSERT INTO pgmetadata.glossary (id, field, code, label_en, description_en, item_order, label_fr, description_fr, label_it, description_it, label_es, description_es, label_de, description_de) VALUES (130, 'dataset.license', 'dl-de/by-2-0', 'Data licence Germany – attribution – version 2.0', NULL, 80, NULL, NULL, NULL, NULL, NULL, NULL, 'Datenlizenz Deutschland – Namensnennung – Version 2.0', NULL);
+INSERT INTO pgmetadata.glossary (id, field, code, label_en, description_en, item_order, label_fr, description_fr, label_it, description_it, label_es, description_es, label_de, description_de) VALUES (131, 'dataset.license', 'proj', 'restricted use for project-related work', NULL, 90, NULL, NULL, NULL, NULL, NULL, NULL, 'nur für Projektbearbeitung', NULL);
```
Comment (Gustry) [inline pg_metadata/install/sql/pgmetadata/90_GLOSSARY.sql:35]:
```suggestion
INSERT INTO pgmetadata.glossary (id, field, code, label_en, description_en, item_order, label_fr, description_fr, label_it, description_it, label_es, description_es, label_de, description_de) VALUES (131, 'dataset.license', 'proj', 'Restricted use for project-related work', NULL, 90, NULL, NULL, NULL, NULL, NULL, NULL, 'nur für Projektbearbeitung', NULL);
```

**Review by Gustry — APPROVED** [review]
Thanks

File: pg_metadata/install/sql/upgrade/upgrade_to_1.1.0.sql (hunk 1 above)
Comment (effjot) [reply pg_metadata/install/sql/upgrade/upgrade_to_1.1.0.sql:210]: Yes, thanks for the correction of the Englisch version

**Pull request merged**